bitwarden_license/src/app/providers/providers-layout.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+    Component,
+    NgZone,
+} from '@angular/core';
+
+import { ActivatedRoute } from '@angular/router';
+
+import { UserService } from 'jslib-common/abstractions/user.service';
+
+import { Provider } from 'jslib-common/models/domain/provider';
+
+@Component({
+    selector: 'providers-layout',
+    templateUrl: 'providers-layout.component.html',
+})
+export class ProvidersLayoutComponent {
+
+    provider: Provider;
+    private providerId: string;
+
+    constructor(private route: ActivatedRoute, private userService: UserService) { }
+
+    ngOnInit() {
+        document.body.classList.remove('layout_frontend');
+        this.route.params.subscribe(async params => {
+            this.providerId = params.providerId;
+            await this.load();
+        });
+    }
+
+    async load() {
+        this.provider = await this.userService.getProvider(this.providerId);
+    }
+
+    get showMenuBar() {
+        return true; // TODO: Replace with permissions
```

Comment:
Do we have a card on the Asana board for this `TODO`?

Reply:
I don't think so. It's a remnant from `OrganizationLayoutComponent`. I'll create an Asana ticket for this.